### ISSUE TITLE
[HOLD for payment 2024-04-25] [POLISH][Members] Address issues with members profile

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#37927]
Coming from [this PR](https://github.com/Expensify/App/pull/37715), we have a few small issues to fix.

1. Navigate to global profile if not an admin of paid policy. We shouldn't just early return [here](https://github.com/Expensify/App/blob/18d88eff913698bbac32889fa20e3cdb286b4079/src/pages/workspace/WorkspaceMembersPage.tsx#L265-L267), instead let's navigate the admin of a free policy to the global profile, like we do [above](https://github.com/Expensify/App/blob/18d88eff913698bbac32889fa20e3cdb286b4079/src/pages/workspace/WorkspaceMembersPage.tsx#L260-L263).

2. Can't see owner's profile. We should probably just navigate to the global profile here.

https://github.com/Expensify/App/assets/22219519/516ef386-02d9-4f97-b8fe-7accdeac7d3e

3. Info icon has active green color by default. We need to fix this.
<img width="663" alt="Screenshot 2024-03-07 at 2 56 15 PM" src="https://github.com/Expensify/App/assets/22219519/adf62631-ec31-4333-8466-4afb51790fe9">

4. https://github.com/Expensify/App/issues/37941
5. https://github.com/Expensify/App/issues/37940 - Backend PR [here](https://github.com/Expensify/Web-Expensify/pull/41227)

cc @burczu 

<details><summary>Issue Owner</summary>Current Issue Owner: @Santhosh-Sellavel</details>

**Comment by burczu** [comment]
@kadiealexander @luacmartins Please assign this issue to me - I'll take care of it as na author of the original issue. Thank you.

**Comment by burczu** [comment]
@luacmartins Ad. 2:

> 2. Can't see owner's profile. We should probably just navigate to the global profile here.

This is intended and I'm addressing it here: https://github.com/Expensify/App/issues/35716 - for owners we will show the Members Details page (if another admin selects it) but with the button that will allow to change the ownership.

**Comment by burczu** [comment]
>Info icon has active green color by default. We need to fix this.
Screenshot 2024-03-07 at 2 56 15 PM

@shawnborton Could you please confirm this is a bug and we need to change the color of the Icon here? I recall we had similar situation with Bulk Actions - icons were grey on the design and green in the existing component...

**Comment by burczu** [comment]
> > Info icon has active green color by default. We need to fix this.
> 
> If we change this, we should globally change all of our option rows. It is something we've talked about given that now the rows in places like Settings don't use green icons anymore. So yeah, definitely open to this but only if we do it as a global change across all option rows/menu rows. Thoughts @Expensify/design ?

Hmmm, so this sounds like a separate issue? @luacmartins

**Comment by luacmartins** [comment]
Oh so the green icon is intentional? I thought it was a bug since we apply that style on hover to icons elsewhere, e.g. workspace settings page in the LHN

**Comment by shawnborton** [comment]
It's intentional, yup. But we should update it to match the the rows in the Settings page, and then make sure we update the hover styles as well.

**Comment by luacmartins** [comment]
@burczu we also have this issue - https://github.com/Expensify/App/issues/37940

**Comment by burczu** [comment]
@luacmartins This is caused by the backend issue we've already discussed [on Slack](https://callstack-hq.slack.com/archives/C06D1M59WG7/p1709730585017929).

**Comment by luacmartins** [comment]
Added https://github.com/Expensify/App/issues/37941 to the scope of this issue.

**Comment by trjExpensify** [comment]
Are we waiting on something to keep this issue open?

**Comment by burczu** [comment]
@trjExpensify I'm still working on some polishing of the Members page, that are related to this issue (back button issues mostly), so it should be still open.

**Comment by burczu** [comment]
@luacmartins I mean this issue: https://github.com/Expensify/App/issues/37941

**Comment by luacmartins** [comment]
Ah ok. It seems like we have a draft PR. What's the ETA to get it in review?

**Comment by trjExpensify** [comment]
Last PR for this merged, right? https://github.com/Expensify/App/pull/37961

**Comment by luacmartins** [comment]
Yes

**Comment by luacmartins** [comment]
Waiting on deploy to prod to process payment

**Comment by kadiealexander** [comment]
Reassigning for someone to handle payment as I'll be OOO!

**Comment by kadiealexander** [comment]
Payouts due:

- [ ] Contributor+: $500 @Santhosh-Sellavel 

No Upwork job, payment via manual requests. Santhosh please request payment once the checklist is done. Thanks!

**Comment by kadiealexander** [comment]
@Santhosh-Sellavel please complete the checklist!

**Comment by luacmartins** [comment]
This was part of a new feature, so I don't think the checklist applies here. Are we good to close this issue?

**Comment by kadiealexander** [comment]
Oh, the `bug` label was applied. Closing, please apply for payment via newdot.

**Comment by Santhosh-Sellavel** [comment]
Requested on ND

**Comment by JmillsExpensify** [comment]
$250 approved for @Santhosh-Sellavel
